llvm/lib/Target/AMDGPU/AMDGPUPostLegalizerCombiner.cpp

```diff
@@ -334,6 +336,19 @@ bool AMDGPUPostLegalizerCombinerImpl::matchRcpSqrtToRsq(
   return false;
 }
 
+void AMDGPUPostLegalizerCombinerImpl::applyOneFDivSqrtToRsq(
+    MachineInstr &MI, const Register &X) const {
+  // B.setInstrAndDebugLoc(MI);
+
+  Register Dst = MI.getOperand(0).getReg();
+
+  B.buildIntrinsic(Intrinsic::amdgcn_rsq, ArrayRef<Register>({Dst}))
```

Comment:
```suggestion
  B.buildIntrinsic(Intrinsic::amdgcn_rsq, {Dst})
```
This should work I think, maybe you don't even need the `{}` - I don't remember if implicit ctor works in this case